EmojiArtUITests/EmojiArtUITests.swift

```diff
@@ -10,30 +10,41 @@ import XCTest
 class EmojiArtUITests: XCTestCase {
 
     override func setUpWithError() throws {
-        // Put setup code here. This method is called before the invocation of each test method in the class.
-
-        // In UI tests it is usually best to stop immediately when a failure occurs.
+        super.setUp()
         continueAfterFailure = false
-
-        // In UI tests it’s important to set the initial state - such as interface orientation - required for your tests before they run. The setUp method is a good place to do this.
     }
 
     override func tearDownWithError() throws {
-        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        super.tearDown()
     }
 
-    func testExample() throws {
-        // UI tests must launch the application that they test.
+    func testEditDocumentTitle() throws {
         let app = XCUIApplication()
         app.launch()
+        
+        if app.navigationBars["Untitled"].exists {
+            app.navigationBars["Untitled"]/*@START_MENU_TOKEN@*/.buttons["BackButton"]/*[[".buttons[\"Emoji Art\"]",".buttons[\"BackButton\"]"],[[[-1,1],[-1,0]]],[0]]@END_MENU_TOKEN@*/.tap()
+        }
+        let emojiArtNavigationBar = app.navigationBars["Emoji Art"]
+        emojiArtNavigationBar.buttons["Edit"].tap()
+        app.tables.textFields["Untitled"].tap()
+        app.tables.textFields["Untitled"].doubleTap()
+//        app.keys["N"].tab()
```

Comment:
Try to never commit comment code unless it's a Work in Progress branch or similar